### ISSUE TITLE
Chromium low memory warning

### ISSUE THREAD

**Issue by entozoon** [MichaIng/DietPi#7248]
Running chromium on say, a raspberry pi 3 model A in kiosk mode kicks out a warning dialog that you can't prevent.

<img width="657" alt="Screenshot 2024-10-18 at 10 10 37 AM" src="https://github.com/user-attachments/assets/4b350189-174c-4e7a-96f4-8e4ceef4fe56">

I found out it, [with some help](https://issues.chromium.org/issues/373891800#comment5), it's the `/usr/bin/chromium-browser` script that does it.

Does this script come from DietPi? Or is it from the [RPi-Distro debian script](https://github.com/RPi-Distro/chromium/blob/6b522e37f4bcb41fa8aafc5cf9e8ed913d324395/debian/scripts/chromium#L59)? I can see it's the same.

Either way I'd recommend gouging out this warning because, well, not only does it run reasonably well depending on your purposes, but it shouldn't have interstitial dialogs when booting into kiosk mode.

**Comment by Joulinar** [comment]
we don't build Chromium ourselves. This is the default from official Debian/RPI source

**Comment by entozoon** [comment]
Okay cool, I'll move my issue over there. Thank you

**Comment by Django1982** [comment]
You can supress this message, when you set "want_memcheck=0" in the mentioned file from entozoon.
In my case this setting is on line 122.

**Comment by entozoon** [comment]
Yeah or a `--no-memcheck` flag as mentioned in the [RPi-Distro issue](https://github.com/RPi-Distro/chromium/issues/51#issuecomment-2422232700) (that was also closed, thanks to the tip

**Comment by MichaIng** [comment]
It is an RPi-only warning. I do not find out reasonably at all, hence we'll disable it by default on DietPi: https://github.com/RPi-Distro/chromium/commit/7f411d6ad53c1e8be8f6b864341ec27772826442
